Tests/Unit/MSFT_xIISFeatureDelegation.tests.ps1

```diff
@@ -10,40 +10,48 @@ if (($repoSource -ne $null) -and ($repoSource[0].RepositorySourceLocation.Host -
 {
     if ( -not (Test-Path -Path (Join-Path -Path $moduleRoot -ChildPath 'Tests\DscResourceTestHelper')) )
     {
-
+        # Module was installed from the gallery - so user does not need git in order to run tests
         $choice = read-host "In order to run this test you need to install a helper module, continue with installation? (Y/N)"
 
         if ($choice -eq "y"){
-            #install test folders from gallery
+            # Install test folders from gallery
             Save-Module -Name 'DscResourceTestHelper' -Path (Join-Path -Path $moduleRoot -ChildPath 'Tests')
         }
     
         else 
         {
-            Write-Host "Unable to run tests without the required helper module - Exiting test"
+            Write-Error "Unable to run tests without the required helper module - Exiting test"
             return
         }
         
     }
-
-    $testModule = Get-Module -Name 'DscResourceTestHelper' -ListAvailable
-    $testModuleVer = $testModule[0].Version
+    Push-Location
+    cd (Join-Path -Path $moduleRoot -ChildPath '\Tests\DscResourceTestHelper')
+    $verFolder = Get-ChildItem
```

Comment:
Also considering we're just getting the contents could we just do

````powershell
Get-ChildItem -Path (Join-Path -Path $moduleRoot -ChildPath '\Tests\DscResourceTestHelper')
````

Instead of pushing/changing directories/popping?